interface/src/app/node/node.js

```diff
@@ -5,7 +5,10 @@
 angular.module('adage.node', [
   'ui.router',
   'ui.bootstrap',
-  'ngResource'
+  'ngResource',
+  // Dependency modules from the 'adage' Angular app
+  'adage.utils'
```

Comment:
Not sure if `adage.utils` or a specific stats package would make more sense. Worried that we'll use these again and again. At least the hypergeometric probably will be [if that's in `adage.utils` now].